tgui/packages/tgui/interfaces/Terminal.js

```diff
@@ -0,0 +1,71 @@
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+import { Box, Input, Button, Section } from 'tgui/components';
+import { ColorBox, Flex } from '../components';
+
+export const Terminal = (props, context) => {
+  const { act, data } = useBackend(context);
+  const peripherals = data.peripherals || [];
+  const { textInput } = props;
+
+  const {
+    fdisk,
+    idcard,
+    temp,
```

Comment:
you could not call this `temp`, but the fact that the entire terminal system works on one var called `temp` might be a piece of history at this point